lib/jobs/run-sku-graph.js

```diff
@@ -87,29 +87,29 @@ function runSkuGraphJobFactory(
                 return;
             }
 
-            return waterline.skus.needOne({ id: node.sku });
-        })
-        .then(function(sku) {
-            var graphName = sku.discoveryGraphName;
-            var graphOptions = sku.discoveryGraphOptions || {};
-            // If we don't specify the instanceId in the options
-            // then we can't track when the graph completes
-            graphOptions.instanceId = self.graphId;
+            return waterline.skus.needOne({ id: node.sku })
+            .then(function(sku) {
```

Comment:
The previous code is flattened and that's the root cause, so I intentionally change it to nesting mode.